### ISSUE TITLE
Fail to add external packages 

### ISSUE THREAD

**Issue by pkieltyka** [kardianos/govendor#86]
For reasons I dont know, govendor got stuck at the error:

```shell
$ govendor add +external
Error: Package "github.com/pressly/omnibed/vendor/github.com/golang/protobuf/proto" already in vendor.
```

See https://gist.github.com/pkieltyka/59b31d93f42a0d96fade for some console output. The only way I could move past this is to fork and patch by skipping the error and allowing the program to continue  https://github.com/pkieltyka/govendor/commit/37312afdb04c5fe3d33979d70fb37deae87c2b0e

**Comment by kardianos** [comment]
I agree this is an issue. I know roughly what is causing it and will have a resolution soon.

**Comment by pkieltyka** [comment]
@kardianos unfortunately I'm still getting this error. .. see console output:

```shell
[peter@pak ~/Dev/go/src/github.com/kardianos]$ cd govendor 
[peter@pak ~/Dev/go/src/github.com/kardianos/govendor]$ git pull
remote: Counting objects: 19, done.
remote: Compressing objects: 100% (19/19), done.
remote: Total 19 (delta 9), reused 0 (delta 0), pack-reused 0
Unpacking objects: 100% (19/19), done.
From github.com:kardianos/govendor
   77090b8..2ac4e85  master     -> origin/master
 * [new branch]      direct     -> origin/direct
Updating 77090b8..2ac4e85
Fast-forward
 README.md | 11 ++++++++++-
 run.go    | 11 +++++++----
 2 files changed, 17 insertions(+), 5 deletions(-)
[peter@pak ~/Dev/go/src/github.com/kardianos/govendor]$ go install .
[peter@pak ~/Dev/go/src/github.com/kardianos/govendor]$ z omnibed
[peter@pak ~/Dev/go/src/github.com/pressly/omnibed]$ rm -rf vendor
[peter@pak ~/Dev/go/src/github.com/pressly/omnibed]$ govendor init
[peter@pak ~/Dev/go/src/github.com/pressly/omnibed]$ govendor add +external
Error: Failed to copy package "/Users/peter/Dev/go/src/github.com/stretchr/testify/vendor/github.com/davecgh/go-spew/spew/" -> "/Users/peter/Dev/go/src/github.com/pressly/omnibed/vendor/github.com/davecgh/go-spew/spew": open /Users/peter/Dev/go/src/github.com/pressly/omnibed/vendor/github.com/stretchr/testify/vendor/github.com/davecgh/go-spew/LICENSE: no such file or directory
[peter@pak ~/Dev/go/src/github.com/pressly/omnibed]$ govendor add +external
Error: Package "github.com/pressly/omnibed/vendor/github.com/golang/protobuf/proto" already in vendor.
```

**Comment by pkieltyka** [comment]
you'll notice there are 2 errors there though, the first for some reason fails to copy a package, but a second attempt of running add +external gets past that error, and runs up against another bug.

**Comment by kardianos** [comment]
ah, different error. I'll look into the license copy error.

**Comment by kardianos** [comment]
This should be fixed (someone filed issue #88, which is the issue).

I'm not confident it fixed the second error, but I'd like to do one at a time. Would you mind re-testing?

**Comment by pkieltyka** [comment]
Bingo :)
